stdlib/jdbc/src/main/ballerina/src/java.jdbc/types.bal

```diff
@@ -25,12 +25,15 @@ import ballerinax/java;
 # + poolOptions - Properties for the connection pool configuration. Refer `PoolOptions` for more details
 # + dbOptions - A map of DB specific properties. These properties will have an effect only if the dataSourceClassName is
 #               provided in poolOptions
+# + getGeneratedKeys - Allow to retrieves auto-generated keys after a statement has been executed if value is `true`.
+#                      It will return auto-generated keys only if supported by JDBC
 public type ClientConfiguration record {|
     string url = "";
     string username = "";
     string password = "";
     PoolOptions poolOptions?;
     map<anydata> dbOptions = {};
+    boolean getGeneratedKeys = false;
```

Comment:
Why this is a property in ClientConfiguration level? Can't we make it a optional parameter in update action only?